addon/templates/components/polaris-text-field/resizer.hbs

```diff
@@ -5,7 +5,29 @@
   This is because of the `white-space: pre-wrap` style applied by Polaris,
   which replicates the white space/wrapping behaviour of the textarea control.
 --}}
-<div class="Polaris-TextField__DummyInput">{{finalContents}}</div>
-{{#if minimumLines}}
-  <div class="Polaris-TextField__DummyInput">{{contentsForMinimumLines}}</div>
-{{/if}}
+{{!-- template-lint-disable no-invalid-interactive --}}
+<div
+  class="Polaris-TextField__Resizer"
+  aria-hidden="true"
+  data-test-text-field-resizer
+  ...attributes
+  {{on "resize" this.handleHeightCheck}}
+  {{did-insert this.handleHeightCheck}}
+  {{did-update this.handleHeightCheck @contents @currentHeight @minimumLines}}
+>
+  <div
+    class="Polaris-TextField__DummyInput"
+    {{did-insert this.this.setContentNode}}
+  >
+    {{this.finalContents}}
+  </div>
+
+  {{#if @minimumLines}}
+    <div
+      class="Polaris-TextField__DummyInput"
+      {{did-insert this.setMinimumLinesNode}}
+    >
+      {{this.contentsForMinimumLines}}
+    </div>
```

Comment:
Have you checked that multiline text fields render/resize to fit their content correctly with these changes? There's a comment at the top of this file that says these two dummy inputs need to be on a single line to render the correct size. If that's no longer the case, can we remove the comment?

Reply:
checked now, fixed some issues 🙌